index.php

```diff
@@ -38,10 +38,10 @@ function get_response($key) {
     return $codes_and_values[$key];
 }
 
-function return_key() {
-    global $codes_and_values;
-    return key($codes_and_values);
-}
+# function return_key() {
```

Comment:
Thanks for catching this! Would you mind removing the function altogether in this PR?

Reply:
I removed the function and force-pushed again. Please check if this works for you.